src/Components/Form/SignUp/Form.tsx

```diff
@@ -0,0 +1,118 @@
+        const { value, name, type } = event.target
+        // console.log("name - " + name + "value - " + value);
+        setInputs(values => ({ ...values, [name]: type === 'checkbox' ? !inputs["checked"] : value }))
+    }
+
+
+    const handleSubmit = (event: React.FormEvent<HTMLFormElement>) => {
+        event.preventDefault();
+        console.log(inputs)
+    }
+    return (
+        <div className='page-content'>
+            <main className='signup-page'>
+                <div className='page-heading'>
+                    <h1>Create an Account</h1>
+                    <p>Already have an account? sign in</p>
+                </div>
+                <section className='form-section'>
+                    <div className='form-container'>
+                        <form onSubmit={handleSubmit}>
+                            <label>Name:</label>
+                                <input
+                                    type="text"
+                                    name="name"
+                                    value={inputs.name || ""}
+                                    onChange={(event) => handleChange(event)}
+                                />
+                            <label>DOB:</label>
+                                <input
+                                    type="date"
+                                    name="dob"
+                                    value={inputs.dob}
+                                    onChange={(event) => handleChange(event)}
+                                />
+
+                            <label>Email:</label>
+                                <input
+                                    type="email"
+                                    name="email"
+                                    value={inputs.email || ""}
+                                    onChange={(event) => handleChange(event)}
+                                />
+
+                            <label>Password:</label>
+                                <input
+                                    type="password"
+                                    name="password"
+                                    value={inputs.password || ""}
+                                    onChange={(event) => handleChange(event)}
+                                />
+
+                            <label>Sate:</label>
+                                <select name="state"
+                                    onChange={(event) => handleChange(event)}
+                                    value={inputs.state}>
+                                    <option value=""></option>
+                                    <option value="maharashtra">Maharashtra</option>
+                                    <option value="gujrat">Gujrat</option>
+                                    <option value="karnatak">Karnatak</option>
+                                    <option value="rajasthan">Rajasthan</option>
+                                </select>
+
```

Comment:
declare const and move all the values in that and add the map function to render out the data.

Eg const stateName=[{value:"gujarat",label:"Gujarat"}]